src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.Linker.targets

```diff
@@ -47,4 +47,13 @@ This file contains the .NET 5-specific targets to customize ILLink
       DependsOnTargets="_ResolveAssemblies;_CreatePackageWorkspace;$(_BeforeLinkAssemblies);_GenerateJniMarshalMethods;_LinkAssembliesNoShrink"
   />
 
+  <Target Name="_TouchAndroidLinkFlag"
+      AfterTargets="ILLink"
+      Condition=" '$(PublishTrimmed)' == 'true' and Exists('$(_LinkSemaphore)') "
+      Inputs="$(_LinkSemaphore)"
+      Outputs="$(_AndroidLinkFlag)">
+    <!-- This file is an input for _RemoveRegisterAttribute -->
+    <Touch Files="$(_AndroidLinkFlag)" AlwaysCreate="true" />
+  </Target>
```

Comment:
I couldn't find a way other than to use `AfterTargets="ILLink"`: 

https://github.com/dotnet/sdk/blob/e448c2bd3482d0dbd9bab06de589e85db5f47e56/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.targets#L34-L37